test/functional/server-client.test.js

```diff
@@ -406,6 +410,49 @@ test('proxy requests originating from behind the broker server', (t) => {
         );
       });
 
+      t.test('successfully broker POST to git client', (t) => {
+        const url = `http://localhost:${serverPort}/broker/${token}/snykgit/echo-body`;
+        const body = { some: { example: 'json' } };
+        request({ url, method: 'post', json: true, body }, (err, res) => {
+          t.equal(res.statusCode, 200, '200 statusCode');
+          t.same(res.body, body, 'body brokered');
+          t.end();
+        });
+      });
+
+      t.test('successfully broker exact bytes of POST body', (t) => {
```

Comment:
should mention `git client` in this and the other tests related to that flow.

Also I'd change `broker` to `redirects` because it makes more sense to me (feel free to break that convention)

Reply:
@carlos-snyk Great suggestions, let me implement those as a last step! Thanks for review 🙏🏻